### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by karliss**

### CODE REVIEW

**Review by ITAYC0HEN — APPROVED** [review]

**Comment by karliss** [comment]
@ITAYC0HEN PRs from `l10n_master` are automatically generated and updated by Crowdin. They don't need to be merged instantly. Once in a while and before release is fine. They look like they are made by me because Github permission model is awful.

**Comment by ret2libc** [comment]
I guess I can/should merge this now, right?

File: zh-CN/cutter_zh.ts [hunk 1]
```diff
@@ -3623,7 +3623,7 @@ WARNING: In write mode any changes will be commited to disk</source>
     <message>
       <location filename="../widgets/FlagsWidget.ui" line="113"/>
       <source>Del</source>
-      <translation>删除</translation>
+      <translation>Del</translation>
```
Comment (Maijin) [inline zh-CN/cutter_zh.ts:3244]: This looks like a regression no?

**Comment by xarkes** [comment]
@ret2libc I guess we can, unless as Maijin mentioned there are regressions...
I think we would need to check the status on Crowdin and see if the synchronization is actually working.

**Comment by ret2libc** [comment]
> @ret2libc I guess we can, unless as Maijin mentioned there are regressions...
> I think we would need to check the status on Crowdin and see if the synchronization is actually working.

I have no idea how this thing works though :( could you or someone familiar with translation check this?

**Pull request merged**